ext/serializer.c

```diff
@@ -806,7 +812,7 @@ static void _serialize_meta(zval *el, ddtrace_span_data *span) {
     meta = &meta_zv;
 
     zval *exception_zv = ddtrace_spandata_property_exception(span);
-    if (Z_TYPE_P(exception_zv) == IS_OBJECT && instanceof_function(Z_OBJCE_P(exception_zv), zend_ce_throwable)) {
+    if (!ignore_error && Z_TYPE_P(exception_zv) == IS_OBJECT && instanceof_function(Z_OBJCE_P(exception_zv), zend_ce_throwable)) {
```

Comment:
In fact, I would propose, if an error was ignored, and another exception is thrown, it will not be ignored. I.e.:
```suggestion
    if (Z_TYPE_P(exception_zv) == IS_OBJECT && instanceof_function(Z_OBJCE_P(exception_zv), zend_ce_throwable)) {
        ignore_error = false;
```